test/run_workflows/step_parameters_test.exs

```diff
@@ -0,0 +1,102 @@
+defmodule StepFlow.RunWorkflows.StepParametersTest do
+  use ExUnit.Case
+  use Plug.Test
+
+  alias StepFlow.WorkflowDefinitions.WorkflowDefinition
+
+  doctest StepFlow
+
+  describe "workflows" do
+    @workflow_definition %{
+      identifier: "id",
+      version_major: 6,
+      version_minor: 5,
+      version_micro: 4,
+      reference: "some id",
+      icon: "custom_icon",
+      label: "Parallel steps with notification",
```

Comment:
Maybe something closer to the name of the test? "Workflow with all the possible types"